hkmc2/shared/src/test/mlscript/OverloadedModulesInSignatures.mls

```diff
@@ -33,7 +33,7 @@ fun f: module TypMod = TypMod
 
 :e
 fun f: TrmMod = 42
-//│ ╔══[ERROR] Expected a non-module type; found reference denoting module definition 'TrmMod'.
+//│ ╔══[ERROR] Expected a non-module type; found reference.
```

Comment:
This message doesn't make much sense :/

Reply:
Anyway, let's stop spending time on module checks

Reply:
Yeah I see. I am fixing it

Reply:
OK this is because that the error occurs during type resolution, so at the time the type is not ready. I changed the old message back for errors occurring during type resolution.